MachineLearning_Engine/Compute/InstallMachineLearningToolkit.cs

```diff
@@ -0,0 +1,87 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2020, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.oM.Reflection;
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace BH.Engine.MachineLearning
+{
+    public static partial class Compute
+    {
+        /*************************************/
+        /**** Public Methods              ****/
+        /*************************************/
+
+        public static Output<bool, List<string>> InstallMachineLearningToolkit(bool run = false, bool force = false)
```

Comment:
I'm being a PITA here, but could we get a description + input/output attributes for this? Just to help people? Also could the description give a warning that it may take time to complete (similar to the Python one)?

Reply:
I had to do it: https://en.wikipedia.org/wiki/Pita